src/PoCs/KafkaStatusPopover.tsx

```diff
@@ -0,0 +1,98 @@
+import { Popover, ProgressStep, ProgressStepper, Button } from "@patternfly/react-core";
+import React, {
+  FunctionComponent,
+  useState,
+  VoidFunctionComponent,
+} from "react";
+ 
+type KafkaStatusPopoverProps = {
+  shouldStartOpen?: boolean;
+} & KafkaStatusPopoverBodyProps;
+
+export const KafkaStatusPopover: FunctionComponent<KafkaStatusPopoverProps> = ({
+  shouldStartOpen = false,
+  currentState,
+  children,
+}) => {
+  const [isVisible, setIsVisible] = useState(shouldStartOpen);
+
+  return (
+    <Popover
+      headerContent="Creating instance"
+      bodyContent={<KafkaStatusPopoverBody currentState={currentState} />}
+      isVisible={isVisible}
+      shouldOpen={() => setIsVisible(true)}
+      shouldClose={() => setIsVisible(false)}
+    >
+      {children}
+    </Popover>
+  );
+};
+
+type States = "pending" | "provisioning" | "preparing" | "ready" ;
+
+type KafkaStatusPopoverBodyProps = {
+  currentState: States;
+};
+export const KafkaStatusPopoverBody: VoidFunctionComponent<
+  KafkaStatusPopoverBodyProps
+> = ({ currentState }) => {
+  const order: States[] = ["pending", "provisioning", "preparing", "ready"];
+  const currentStep = order.findIndex((v) => v === currentState);
+  console.log(currentStep);
+  return (
+    <div>
+      <p>This will be ready shortly. In the meantime you can create a service account under the <a href="http://www.google.com/">Connections tab.</a> </p>
```

Comment:
```suggestion
      <p>This will be ready shortly. In the meantime, you can create a service account under the <a href="http://www.google.com/">Connections tab.</a> </p>
```